x-pack/plugins/ml/common/constants/messages.ts

```diff
@@ -626,6 +626,30 @@ export const getMessages = once((docLinks?: DocLinksStart) => {
           'the UNIX epoch beginning. Timestamps before 01/01/1970 00:00:00 (UTC) are not supported for machine learning jobs.',
       }),
     },
+    datafeed_preview_no_documents: {
+      status: VALIDATION_STATUS.WARNING,
+      heading: i18n.translate(
+        'xpack.ml.models.jobValidation.messages.datafeedPreviewNoDocumentsHeading',
+        {
+          defaultMessage: 'Datafeed preview',
+        }
+      ),
+      text: i18n.translate(
+        'xpack.ml.models.jobValidation.messages.datafeedPreviewNoDocumentsMessage',
+        {
+          defaultMessage:
+            'Running the datafeed preview over the current job configuration produces no results.' +
+            'If the index contains no documents this warning can be ignored, otherwise the job may be misconfigured.',
```

Comment:
Nit: Noticed in the screenshot concatenating like that ends up without a space between `.` and `If`.

Reply:
Fixed in https://github.com/elastic/kibana/pull/109080/commits/b8cff0b7aeda6ff54c2101e9a0ac8d800b7c00a4